content/tutorials/editors.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Recommended Editors
+---
+
+To edit your code, you'll need an editor.
+
+{{% notice info %}}
+To write code for the kit, you don't need anything specific in your editor, you just need to be able to edit Python (`.py`) files, and get them onto a USB drive.
+{{% /notice %}}
+
+Any editor will do, so it's recommended to use one you're already familiar with. If you don't already have a preference, here's some great ones, available on Windows, macOS, and Linux!
+
+- [Sublime Text 3](https://sublimetext.com)
```

Comment:
Sublime is (strictly speaking) a paid-for editor, for any use. I'd suggest that we move it down the list (or note that you're meant to buy it)